src/components/AccordionMenu1/AccordionMenu1.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="accordion-menu-1">
```

Comment:
Why are we not using bootstrap's accordion component? https://bootstrap-vue.js.org/docs/components/collapse#accordion-support